core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/impl/api/parse.kt

```diff
@@ -271,6 +272,13 @@ internal object Parsers : GlobalParserOptions {
         ) else null
         return parser.applyOptions(options)
     }
+
+    internal fun getDoubleConverter(locale: Locale? = null): TypeConverter {
+        val options = if (locale != null) ParserOptions(
+            locale = locale
+        ) else null
+        return parserToDoubleWithOptions.toConverter(options)
```

Comment:
You can use `parserToDoubleWithOptions.applyOptions` here, i think

Reply:
Done. Originally I used `toConverter` for getting TypeConversionException  to be thrown. Now it is thrown in `.convertToDouble` function and this is renamed to `getDoubleParser`.